devicetypes/smartthings/ecobee-sensor.src/ecobee-sensor.groovy

```diff
@@ -68,6 +67,6 @@ def refresh() {
 
 void poll() {
 	log.debug "Executing 'poll' using parent SmartApp"
-	parent.pollChild(this)
+	parent.pollChild()
```

Comment:
@dsainteclaire This doesn't impact the polling at all. Since this is a cloud integration, the API calls can return the data for all the devices in the same API call. So there is no need to pass the information directly related to which child made the call since we will get the data back for all children at the same time.